src/library/scala/collection/immutable/SortedMap.scala

```diff
@@ -18,6 +18,41 @@ import scala.annotation.unchecked.uncheckedVariance
 import scala.collection.mutable.Builder
 import scala.language.higherKinds
 
+
+/** A class for immutable maps whose keys-value pairs are sorted according to a [[scala.math.Ordering]].
+  *
+  * This class is optimal when range queries will be performed, or when traversal in order of an ordering is desired. If
```

Comment:
1. Since this is not a class but a trait, which is to say not an implementation but an interface, we cannot assert that it is _optimal_ for any purpose. Rather, we should say that is _intended_, or that it _allows_ for those purposes.
2. Rather than "in order of an ordering", perhaps we could write "following an order relation between keys". 
3. I would introduce a newline after the `performed,`, and another after `desired.`.